packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -130,171 +126,59 @@ const genLTSConfig = (compatVersion: number | string): CompatConfig[] => {
-                configList.push(...genLTSConfig(lts));
-            });
-        } else {
-            const num = parseInt(value, 10);
-            if (num.toString() === value) {
-                configList.push(...genConfig(num));
-            } else {
-                configList.push(...genConfig(value));
-            }
-        }
-    });
-}
-
-if (compatKind !== undefined) {
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    configList = configList.filter((value) => compatKind!.includes(value.kind));
-}
-
-function createExpectsTest(orderedExpectedEvents: ITelemetryGenericEvent[], test: Mocha.AsyncFunc){
-    return async function (this:Context){
-        const provider: TestObjectProvider | undefined = this.__fluidTestProvider;
-        if(provider === undefined){
-            throw new Error("Expected __fluidTestProvider on this");
-        }
-        try{
-            provider.logger.registerExpectedEvent(... orderedExpectedEvents);
-            await test.bind(this)();
-        }catch(error){
-            // only use TestException if the event is provided.
-            // it must be last, as the events are ordered, so all other events must come first
-            if(orderedExpectedEvents[orderedExpectedEvents.length -1]?.eventName === "TestException"){
-                provider.logger.sendErrorEvent({eventName:"TestException"},error)
-            }else{
-                throw error;
-            }
-        }
-        const err = getUnexpectedLogErrorException(provider.logger);
-        if(err !== undefined){
-            throw err;
-        }
-    };
-}
-
-export type ExpectsTest =
-    (name: string, orderedExpectedEvents: ITelemetryGenericEvent[], test: Mocha.AsyncFunc) => Mocha.Test
-
-/**
- * Similar to mocha's it function, but allow specifying expected events.
- * That must occur during the execution of the test.
- */
-export const itExpects: ExpectsTest & Record<"only" |"skip", ExpectsTest> =
-    (name: string, orderedExpectedEvents: ITelemetryGenericEvent[], test: Mocha.AsyncFunc): Mocha.Test =>
-        it(name, createExpectsTest(orderedExpectedEvents, test));
-
-itExpects.only =
-    (name: string, orderedExpectedEvents: ITelemetryGenericEvent[], test: Mocha.AsyncFunc) =>
-        it.only(name, createExpectsTest(orderedExpectedEvents, test));
 
-itExpects.skip =
-    (name: string, orderedExpectedEvents: ITelemetryGenericEvent[], test: Mocha.AsyncFunc) =>
-        it.skip(name, createExpectsTest(orderedExpectedEvents, test));
+export const configList = new Lazy<readonly CompatConfig[]>(()=>{
 
-
-/*
- * Mocha Utils for test to generate the compat variants.
- */
-function describeCompat(
-    name: string,
-    tests: (provider: () => ITestObjectProvider) => void,
-    compatFilter?: CompatKind[],
-) {
-    let configs = configList;
-    if (compatFilter !== undefined) {
-        configs = configs.filter((value) => compatFilter.includes(value.kind));
+    // set it in the env for parallel workers
```

Comment:
here is all the stuff that just moved. nothing changes in this file besides the Lazy above, everything else was just pulled to the new files